inst/WORDLIST

```diff
@@ -28,6 +29,7 @@ SUPPTR
 SV
 anonymized
 funder
+ggplot
```

Comment:
I think we want ggplot2 not to throw an error for the Spelling check so we should leave it in.

Reply:
@bms63 , specifying "ggplot2" in WORDLIST did not seem to work, but I've now added back ticks around names of packages, so it's no longer necessary to specify "ggplot" or "ggplot2" in the WORDLIST file.